src/System.Net.Ping/tests/FunctionalTests/UnixPingUtilityTests.cs

```diff
@@ -69,6 +78,10 @@ public class UnixPingUtilityTests
             var exitCode = p.ExitCode;
             if (exitCode != 0)
             {
+                // Workaround known issues with ping6 in macOS 10.12
+                if (isPing6OnMacSierra)
+                    return;
```

Comment:
If we're just ignoring all failures, why run the test in this condition at all? Or are there things we're still verifying?

Reply:
Being conservative: per comments in old issues dealing with these failures the wrapper that actually uses ping6 will already raise errors when the execution fails. This change keeps validating the ping6 output when it does succeed - it is unlikely that it will ever change but this way we will know if it does happen. Side note: the previous assumption when trying to deal with these failures was that even under some crashes we could still parse the output, that was too optimistic.